api/src/routes/publications.js

```diff
@@ -32,8 +32,9 @@ publicationsRouter.post(
 
 publicationsRouter.get(
   '/import/:gScholarUserId/:startFrom/validate/:team_id',
-  authMiddleware.cookieJwtAuth,
+  // authMiddleware.cookieJwtAuth,
```

Comment:
please remove if not needed

Reply:
oops, i always forget this. will fix on main